src/index.js

```diff
@@ -209,6 +209,9 @@ function _renderToString(vnode, context, isSvgMode, selectValue, parent) {
 		}
 		return rendered;
 	}
+	
+	// VNodes have {constructor:undefined} to prevent JSON injection:
+	if (vnode.constructor !== undefined) return '';
```

Comment:
can we add a few tests like

```
<div{{ hello: 'world' }}</div>
```

and the one from the issue

Reply:
Yep, will do.